### PR TITLE
Pubblish me.iepure.devtoolbox

### PR DESCRIPTION
**Pull request by aleiepure**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

~~The only extra permission is `--filesystem=/tmp`. The app needs r/w access to `/tmp` for temporary files. Using portals or environment variables leads to errors with GTK's internal URI handling when reading. Full path access is the only working way I could find.~~

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: me.iepure.devtoolbox.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "devtoolbox",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland",
+        "--filesystem=/tmp"
```
Comment (hfiguiere) [inline me.iepure.devtoolbox.json:13]: do you really need this?

**Comment by hfiguiere** [comment]
bot, build me.iepure.devtoolbox

File: me.iepure.devtoolbox.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/aleiepure/devtoolbox",
+                    "tag": "v1.0",
+                    "commit": "0d9c014de0cb3f9186580070183c5b42032ac9a7"
+                }
+            ],
+            "config-opts" : [
```
Comment (hfiguiere) [inline me.iepure.devtoolbox.json:40]: config-opts are usually above sources.

File: me.iepure.devtoolbox.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        "pypi-dependencies.json",
+        {
+            "name" : "devtoolbox",
+            "builddir" : true,
```
Comment (hfiguiere) [inline me.iepure.devtoolbox.json:30]:
```suggestion
```

this is default.

**Comment by aleiepure** [comment]
I found a workaround for the /tmp permission (now removed) and modified the manifest as suggested.

**Comment by aleiepure** [comment]
bot, build me.iepure.devtoolbox

**Comment by aleiepure** [comment]
@hfiguiere any updates?

**Comment by barthalion** [comment]
bot, build me.iepure.devtoolbox

**Pull request closed without merging**